lib/rucio/core/replica.py

```diff
@@ -835,6 +836,12 @@ def _list_replicas(dataset_clause, file_clause, state_clause, show_pfns, schemes
                                     if root_proxy_internal and 'value' in root_proxy_internal[0]:
                                         pfn = root_proxy_internal[0]['value'] + '//' + pfn
 
+                        # do we need to sign the URLs?
+                        if sign_urls and protocol.attributes['scheme'] == 'https':
+                            sign = get_rse_attribute('sign_url_gcs', rse_info[rse]['id'], session=session)
```

Comment:
`sign_url` is more generic wrt s3, swift, etc

Reply:
agreed, changing